tasks/embed-fonts.js

```diff
@@ -22,8 +22,8 @@ module.exports = function (grunt) {
 
   function getDataUri(fontFile) {
     var typeMatch = fontType.exec(fontFile),
-        faceContent = grunt.file.read(fontFile),
-        fontEncoded = new Buffer(faceContent).toString('base64');
+        faceContent = grunt.file.read(fontFile, {encoding: null}),
+        fontEncoded = faceContent.toString('base64');
```

Comment:
I only needed to change line 25

leave as fontEncoded = new Buffer(faceContent).toString('base64');  ??

Or does a new buffer instance eat more memory?